app/src/main/java/org/fossasia/openevent/general/auth/LoginFragment.kt

```diff
@@ -146,4 +147,13 @@ class LoginFragment : Fragment() {
             else -> super.onOptionsItemSelected(item)
         }
     }
+
+    private fun showSnackbar() {
+        val textSnackbar = arguments?.getString("snackbar_message")
```

Comment:
Where is this used below?

Reply:
I only used this to check whether snackbar should be showed or not in login fragment.